website/docs/providers/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Providers
+slug: /providers
+---
+
+A Provider is an authentication method, a service that is used by authentik to authenticate the user for the associated application. Common Providers are OpenID Connect (OIDC)/OAuth2, LDAP, SAML, and generic proxy provider, and others.
+
+Providers are the "other half" of [applications](../applications/index.md). They typically exist in a 1-to-1 relationship; each application needs a provider and every provider can be used with one application.
```

Comment:
we might want to mention back-channel providers here like we do on the applications page